src/pcapi/core/offers/models.py

```diff
@@ -214,6 +214,12 @@ def before_insert(mapper, configuration, self):
 event.listen(Stock.__table__, "after_create", DDL(Stock.trig_update_date_ddl))
 
 
+class OfferImage:
+    def __init__(self, url: str, credit: Optional[str]) -> None:
+        self.url = url
+        self.credit = credit
```

Comment:
En un peu plus concis :

```suggestion
@dataclasses.dataclass
class OfferImage:
    url : str
    credit : Optional[str] = None
```